plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -111,6 +111,7 @@ public final class IcebergSessionProperties
     private static final String INCREMENTAL_REFRESH_ENABLED = "incremental_refresh_enabled";
     public static final String BUCKET_EXECUTION_ENABLED = "bucket_execution_enabled";
     public static final String FILE_BASED_CONFLICT_DETECTION_ENABLED = "file_based_conflict_detection_enabled";
+    private static final String ICEBERG_MAX_PARTITIONS_PER_WRITER = "iceberg_max_partitions_per_writer";
```

Comment:
No need to include the name "iceberg" in the session property name since we're in the context of `IcebergSessionProperties` and the iceberg connector already.